lib/h2.mli

```diff
@@ -502,6 +502,8 @@ module Reqd : sig
     -> Response.t
     -> [ `write ] Body.t
 
+  val send_trailers_on_close : t -> Headers.t -> unit
```

Comment:
nit: I'd prefer if this was rather called `schedule_trailers` or `schedule_trailer_headers`

Reply:
Can you also add a docstring, please?